packages/marketplace/src/styles/components/_grid.scss

```diff
@@ -19,6 +19,9 @@
 
 .marketplace-header__right, .marketplace-header__left {
   display: flex;
+  & > * {
```

Comment:
Can we change this to `div` or something instead of `*`? So it's not as broad.

Reply:
It is only every direct child element so it's mostly wrappers, but sure.